R/ML.NeuralNet.R

```diff
@@ -35,31 +35,61 @@ ML.NeuralNet <- R6Class("ML.NeuralNet",
     ),
   active =
     list(
+      get_file_name = function() {
+        return(private$file_name)
+      }
     ),
   private =
     list(
       hidden = NULL,
-      do.fit = function(X_mat, Y_vals, coef = NULL) {
+      file_name = file.path('output', 'model_NN.rds'),
```

Comment:
I've created a separate directory in which you can store the saved output. Note that this should also work on windows systems.

Reply:
However, more importantly, the use of storing these models will probably not work in the case of the full OSL. The problem is that we are fitting multiple neural networks per training (both for hyperparameters as well as for the conditional density estimation). Storing them in a file will simply overwrite them and only use one of the models (which might not at all work well for that specific case.